example-database/README.md

```diff
@@ -0,0 +1,482 @@
+# Appwrite + VueJS =❤️
+This example is to showcase [Appwrite's JS API](https://github.com/appwrite/sdk-for-js) with [VueJS](https://vuejs.org/) by creating a simple tasks web application where you can add, remove and mark tasks as done.
+
+## Prerequisites
+
+-   A Recent Version of NodeJS
+-   Yarn (Feel free to use NPM if you want to, just switch out the Yarn Commands for their NPM counterparts)
+-   [A locally running appwrite instance](https://appwrite.io/docs/installation).
```

Comment:
Appwrite should start with an uppercase letter

Reply:
@PineappleIOnic This is the only one left. After that I can approve it and can it be

Reply:
Done! 👍